README.md

```diff
@@ -67,6 +67,19 @@ bigflow -h
 
 Read more about [BigFlow CLI](docs/cli.md).
 
+To interact with GCP you need to set a default project and log in:
+
+```shell script
+gcloud config set project <your-gcp-project-id>
+gcloud auth application-default login
+```
+
+Finally, check if your Docker is running:
```

Comment:
If not then what?

Reply:
make it work

Reply:
I mean it's not a docker tutorial and writing "make it work" explicitly sounds harsh :)